lib/utils/http-requestor.js

```diff
@@ -87,7 +87,7 @@ class HttpRequestor extends BaseRequestor {
 
     assert(HookMsgTypes.includes(type));
 
-    const payload = params ? snakeCaseKeys(params, ['customerData', 'sip']) : null;
+    const payload = params ? snakeCaseKeys(params, ['sip']) : null;
```

Comment:
I may be wrong, but I think this change is an error.  The removed code says **not** to modify (by snake-casing) the keys inside customerData.  Which is correct.  Your change, it would seem, leads to exactly the bug we are trying to fix.

Have you verified this bug in your own testing, by sending a createCall POST